bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="mzp-c-hero-image">
+    {{ high_res_img('firefox/accounts/fxa-hero-img.png', {'alt': '', 'width': '1200', 'height': '770'}) }}
+  </div>
+</section>
+
+<div class="mzp-l-content">
+  <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.') }}</h2>
+</div>
+
+<div class="mzp-l-content">
+  {% call feature_card(
+    title=_('Send tabs instantly to your devices'),
+    image_url='firefox/accounts/fxa-send.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-right-half'
+  ) %}
+  <p>{{ _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') }}</p>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Remember your passwords in Lockbox'),
+    image_url='firefox/accounts/fxa-lockbox.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-left-half lockbox-card',
+    link_cta=_('Get the Lockbox App'),
+    link_url='#'
+  ) %}
+    <p>{{ _('No more getting locked out of apps and websites. Lockbox secures all the passwords you’ve saved in Firefox and gives you easy access across all your iOS devices.') }}</p>
+
+    <ul class="mobile-download-buttons mobile-lockbox">
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/kq8zhhz" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Sync your bookmarks everywhere'),
+    image_url='firefox/accounts/fxa-sync.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-right-half bookmarks-card'
+  ) %}
+    <p>{{ _('Bookmark a recipe at home. Check it at the market from your phone. Every bookmark you save is instantly added to all your synced devices.') }}</p>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Put quality content in your Pocket'),
+    image_url='firefox/accounts/fxa-pocket.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-left-half pocket-card',
+    link_cta=_('Get the Pocket App'),
+    link_url='#'
+  ) %}
+    <p>{{ _('Discover the web’s best content, and absorb it anytime – even offline – on any device. Pocket’s listen feature will even read any article aloud to you. All from the Firefox toolbar.') }}</p>
+
+    <ul class="mobile-download-buttons mobile-pocket">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/h5iycst', id='playStoreLink') }}
+      </li>
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/h5iycst" data-link-type="download" data-download-os="iOS">
```

Comment:
The id `appStoreLink` is used here and on line 71  and 161. IDs can only be used once on a  page.